test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1

```diff
@@ -63,6 +63,39 @@ Describe 'Classes inheritance syntax' -Tags "CI" {
         [MyComparable].GetInterface("System.IComparable") | Should -Not -BeNullOrEmpty
     }
 
+    It 'can subclass .NET abstract class' {
+        class TestHost : System.Management.Automation.Host.PSHost
```

Comment:
Can we group related tests in a `Context`? That way we don't need to duplicate the `TestHost` declaration 3 times.